KomaMRIBase/src/KomaMRIBase.jl

```diff
@@ -51,8 +52,8 @@ export NoMotion, SimpleMotion, ArbitraryMotion
 export SimpleMotionType
 export Translation, Rotation, HeartBeat
 export PeriodicTranslation, PeriodicRotation, PeriodicHeartBeat
-export get_spin_coords, sort_motions!
-export LinearInterpolator
+export get_spin_coords, initialize_motion!
+export Interpolator
```

Comment:
I don't think `initialize_motion!` and `Inteprolator` should be exported. We don't want the user to use them.

Reply:
Solved in [f044ede](https://github.com/JuliaHealth/KomaMRI.jl/pull/408/commits/f044edef69c76312a8a47ddf672b7075037dad07) and [9ef5d81](https://github.com/JuliaHealth/KomaMRI.jl/pull/408/commits/9ef5d8166a17807599a9e6ee902d45d516b3562c)